### PR TITLE
Define the administrator role and its rules.

### PR DESCRIPTION
**Pull request by artursouza**
Signed-off-by: Artur Souza <artursouza.ms@outlook.com>

### CODE REVIEW

File: ADMINISTRATORS.md
Comment (msfussell) [inline ADMINISTRATORS.md:9]: Given these duties, this seems like there needs to be a new administrators group in the github that has these privileges. Similar to the release team account. If so, can you add into the steps below that on approval by STC, administrators are added to this account? This is in addition to having access to the STC vault in 1Password for credentials.

File: ADMINISTRATORS.md
Comment (artursouza) [reply ADMINISTRATORS.md:20]: I thought that an admin could also perform the duties to add a new approver or maintainer AFTER STC approval. Any reason not to?

File: ADMINISTRATORS.md
Comment (artursouza) [reply ADMINISTRATORS.md:54]: For people to know who is available at their TZ.

File: ADMINISTRATORS.md
Comment (artursouza) [reply ADMINISTRATORS.md:9]: I agree.

File: ADMINISTRATORS.md
Comment (artursouza) [reply ADMINISTRATORS.md:32]: It is important to determine with type of majority. Simple, 2/3 or 3/5?

File: ADMINISTRATORS.md
Comment (artursouza) [reply ADMINISTRATORS.md:45]: We need to specify which type of majority.

File: ADMINISTRATORS.md
Comment (msfussell) [reply ADMINISTRATORS.md:20]: I read this wrong. I thought this means STC adding. You can ignore this comment.

File: ADMINISTRATORS.md
Comment (yaron2) [reply ADMINISTRATORS.md:32]: Let's use simple

File: ADMINISTRATORS.md
Comment (yaron2) [reply ADMINISTRATORS.md:45]: Agree, need to specify. Let's use simple majority

File: ADMINISTRATORS.md
Comment (msfussell) [reply ADMINISTRATORS.md:32]: Ok. Just feel like this is more complex, given that 3/5 of 7 is 4.2, so is that 4 people out of 7? And 3/5 of 9 is 5.4 so is that 5 people? 3/5 of 11 of 6.6 so this is 7 people. Then this is just a majority in all cases.

File: ADMINISTRATORS.md
Comment (msfussell) [reply ADMINISTRATORS.md:45]: Same as above.

File: ADMINISTRATORS.md
Comment (msfussell) [reply ADMINISTRATORS.md:54]: Ok

File: ADMINISTRATORS.md
Comment (msfussell) [reply ADMINISTRATORS.md:9]: I didn't see any update below on this comment. I suggest creating a github team called `administrators` here https://github.com/orgs/dapr/teams and then saying that this is what makes you and admin as well as access to the STC 1password vault.

Comment (artursouza) [reply ADMINISTRATORS.md:9]: I am adding the text now. GitHub does not allow a Team to be made org owner, only individuals. So, we cannot really create an `administrators` role.

File: ADMINISTRATORS.md
Comment (artursouza) [reply ADMINISTRATORS.md:32]: Done.

File: ADMINISTRATORS.md
Comment (artursouza) [reply ADMINISTRATORS.md:45]: Doing simple.

**Review by yaron2 — APPROVED** [review]

**Pull request merged**
